doc/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 3.1.3)
+project(doc)
+
+find_program(DOXYGEN_PATH doxygen)
+if (NOT DOXYGEN_PATH)
+    message(FATAL_ERROR "Doxygen not found!")
+endif()
+
+set(SOURCES_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../)
+set(DOXY_WORKING_DIR ${CMAKE_CURRENT_BINARY_DIR}/doxygen_doc)
+
+set(PROJECT_NAME "no-OS")
+
+#Source files paths to be included for documentation generation
+set(DOCUMENTED_FILES "${CMAKE_CURRENT_SOURCE_DIR}/projects_page.dox \\
```

Comment:
The .dox files are not present at this point - they are only added by the next commit.

Reply:
You are right. I will swap commits.